linux/setup_repo.sh

```diff
@@ -7,6 +7,26 @@ if [[ ${EUID} -ne 0 ]]; then
      exit 1
 fi
 
+# Check for apt version >= 2.4.0 (if found, assuming Debian-based compatible with repo)
+
+apt --version > /dev/null
+if [[ $? -eq 0 ]]; then
+    APT_VERSION=$(apt --version)
+    APT_MAJOR=$(echo "$APT_VERSION" | cut -d' ' -f 2 | cut -d'.' -f 1)
+    APT_MINOR=$(echo "$APT_VERSION" | cut -d' ' -f 2 | cut -d'.' -f 2)
+    echo "Apt version: ${APT_VERSION}"
+else
+    echo "This script is only compatible with Debian based distributions which have apt, but apt is not available. Please check that your OS is supported."
+    exit 1
+fi;
+
+if (( $(echo "${APT_MAJOR}.${APT_MINOR} < 2.4" | bc -l) )); then
+    echo "Your apt version is incompatible. You cannot install this repository. Please update your OS or use the .deb package from the Website to manually install Jamulus."
+    exit 1
+fi;
+
+# We have an acceptable Apt version, continuing
```

Comment:
I don't think that that's needed.

Reply:
Still open.

Reply:
When you say "I don't think that that's needed" what is _that_ referring to?

Reply:
I mean the comment. It feels random and clear to me.